### DISCUSSION TITLE
⏳ 🔗 Add support for Bluesky in Share Buttons widget

### DISCUSSION THREAD

**Discussion by JoelEadeDesign** [elementor/elementor#29242] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I would like to request adding the Bluesky social media platform to the Share Buttons widget. They have an action intent link that can be used. Reference: [https://docs.bsky.app/docs/advanced-guides/intent-links](url)

### Describe the solution you'd like.

Replicate the current method for Facebook, X (Twitter) etc. share buttons.

### Describe alternatives you've considered.

I've used a shortcode generated by ChatGPT to integrate it with my client's website until Elementor can support it. e.g. [https://tvtonight.com.au/2024/11/music-by-john-williams-2.html](url)

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by rmdes** [comment]
This would be really great ! 
also include a bluesky proper icon for the social widgets

**Comment by creativeaffinity** [comment]
The Bluesky Logo is available in Font Awesome – https://fontawesome.com/search?q=bluesky&o=r

**Reply by rmdes** [reply]
I can't see it when I search from inside elementor, no matter the collection, there isn't any Bluesky icon there...

**Reply by scottgrnt** [reply]
BlueSky is not listed as one of the options in the Share widget. 

Facebook
Twitter
LinkedIn
Pinterest
Reddit
VK
OK
Tumblr
Digg
Skype
StumbleUpon
Mix
Telegram
Pocket
XING
WhatsApp
Email
Print
X
Threads

**Comment by ravenswoodmedia** [comment]
+1

**Comment by caspervoogt** [comment]
Also looking for this. I have been running into this more and more on sites I manage.

**Comment by gdickens** [comment]
I'd also like this feature added. 

Bluesky is extremely popular with the research community, which is the target audience of my elementor websites.

**Comment by rmdes** [comment]
Please?

**Comment by melissadickinson** [comment]
nudge

**Comment by oneepiphany** [comment]
Looking for the same feature update.

**Reply by melissadickinson** [reply]
I was able to solve this on all my various sites by either using the theme share icons (Where available) or Ultimate Addons for Elementor, who added Bluesky a couple months ago.

**Comment by StooHardman** [comment]
Plus one for this, a lot of people are moving from other sites to Blue Sky and it's frustrating not being able to share with a click

**Reply by rmdes** [reply]
It's like Elementor staff has muted this issue and nobody is ever going to see this..really frustrating

**Reply by melissadickinson** [reply]
I was able to solve this on all my various sites by either using the theme share icons (Where available) or Ultimate Addons for Elementor, who added Bluesky a couple months ago.

**Comment by seancswanson** [comment]
We would love to have Blue Sky as an option for our site's audience to share with one click. Any updates on this from Elementor staff?

**Reply by melissadickinson** [reply]
I was able to solve this on all my various sites by either using the theme share icons (Where available) or Ultimate Addons for Elementor, who added Bluesky a couple months ago.

**Reply by Eulenkarawane** [reply]
The solution via Ultimate Addons is not for free, is it?

**Comment by Eulenkarawane** [comment]
I would definitely also want a longer list of supported plattforms!
Right now, the list contains mostly "bad" Social Media plattforms owned by Trump supporting billionars or shady corporations.
I'd like to see Signal, Mastodon, Bluesky there.

@JoelEadeDesign, could you share the shortcode you've generated for your website?
It looks really good!

**Reply by JoelEadeDesign** [reply]
My sincere apologies, @Eulenkarawane . I must have missed this notification. I've come back to this issue as my client asked me about it again today. Over 12 months later and we're still waiting... Thanks Elementor. Putting those millions of revenue to good use, I see :P

Here's my shortcode for adding the BlueSky icon as a shortcode:
https://gist.github.com/JoelEadeDesign/16fbac5d1b24a8ed225151bc3b8d6c9e

**Comment by buckmanhands** [comment]
If you are going to lock us in this walled garden that breaks websites if we stop paying, at least try to keep up. Stop shoving AI at us in every way possible and continue to maintain these simple things. We need Bluesky to continue to work to be able to pay you.
